content/docs/reference/quality-guidelines.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Quality Guidelines
+aliases:
+  - /docs/developing/quality-guidelines
+---
+Keep these guidelines in mind when creating or reviewing a pull request (PR). If you're not sure about a certain guideline when creating a PR, you can submit the PR as a draft and mention that you need help with a specific aspect of your PR. Many developers and community members are willing and able to help your PR improve!
+
+All PRs:
+- should have sound reasoning as to why they should be merged. This should be specified under the "Reason for changes" header when writing the PR description. Even if it seems obvious, others may need the additional context. PRs with little to no reason to be merged are likely to be delayed, ignored, or even closed.
```

Comment:
"Should have _'sound'_ reasoning"? Did you mean "some"?

Reply:
> "Should have _'sound'_ reasoning"? Did you mean "some"?

I believe "sound" is intentional, although it doesn't read well.

Reply:
> > "Should have _'sound'_ reasoning"? Did you mean "some"?
> 
> I believe "sound" is intentional, although it doesn't read well.

Yeah, I haven't heard that definition of "sound".

Reply:
It is "sound" as in "[complete, solid, and secure](https://en.wiktionary.org/wiki/sound#Adjective)"